frappe/public/js/frappe/form/sidebar/form_viewers.js

```diff
@@ -1,3 +1,35 @@
+function openChatWith (viewer) {
```

Comment:
This helper should be in `chat.js`. Also, please don't use cameCase. https://github.com/frappe/erpnext/wiki/Naming-Guidelines